i18n/fr/docusaurus-plugin-content-docs/current/dapps/A-Primer-With-The-Desktop-Wallet.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 10
+title: A primer with the desktop wallet
+---
+
+> ## ⚠️ Cette page n'a pas encore été traduite. Vous pouvez le faire en suivant le lien en pied de page.
```

Comment:
There is a nicer way to display this warning messages. Check out this doc page! https://docusaurus.io/docs/markdown-features/admonitions

```suggestion
:::info
Cette page n'a pas encore été traduite. Vous pouvez le faire en suivant le lien en pied de page.
:::
```

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/1579899/172136744-e0e4f043-5c81-4212-bacd-1be93d4581bc.png">

Reply:
beautiful! thank you!!